public/js/index.js

```diff
@@ -72,8 +87,9 @@ $(function () {
     objNewWidget.position.my = `${$('#widgetPositionMyX').val()} ${$('#widgetPositionMyY').val()}`
     objNewWidget.position.at = `${$('#parentPositionAtX').val()} ${$('#parentPositionAtY').val()}`
     objNewWidget.id = getNextId()
-    console.log(objNewWidget)
+    //console.log(objNewWidget)
     createWidget(objNewWidget, objSocket)
+    positionWidgets()
   }
   // setup the new widget dialog
```

Comment:
I think it's obvious what line 95 does, so this comment can be removed.

Reply:
ok, removed, not sure what benefit we have from removing comments. sometimes I find them helpful to read sections

Reply:
Comments can be very informative, when they add something to the code, either why it was done this way or how it works or where else it's relevant. When a comment